weatherbot/puller.py

```diff
@@ -7,3 +9,27 @@ def get_new(updates, update_id):
         if should_add(item, update_id):
             buffer.append(item)
     return buffer
+
+"""In class Puller we create deque() object. 
```

Comment:
1. You need to write PyDoc comments *below* class declaration (first line): https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
2. Good comment describe why does one need this entity, not [what it is](https://i.redd.it/iuy9fxt300811.png) or what it does: https://realpython.com/python-comments-guide/#python-commenting-best-practices